### PR TITLE
#512: Test for exception on http://www.0crat.com/?PsByFlag=PsGithub

### PR DESCRIPTION
**Pull request by paulodamaso**
For #512:
- Added test for `GET http://www.0crat.com/?PsByFlag=PsGithub&code=1257b773ba07221e7eb5`
- Added test for `GET http://www.0crat.com/?PsByFlag=PsGithub`
- Left todo for fixes.

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#1602](https://github.com/zerocracy/farm/issues/1602) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#1602](https://github.com/zerocracy/farm/issues/1602) is assigned to @krzyk[/z](https://www.0crat.com/u/krzyk), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/2282037603); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: src/test/java/com/zerocracy/tk/TkAppTest.java [hunk 1]
```diff
@@ -74,4 +75,37 @@ public void rendersHomePageViaHttp() throws Exception {
         );
     }
 
+    // @todo #512:30min 0crat is displaying an error page when trying to use
+    //  PsByFlag=PsGithub parameter. This test is triggering this behavior.
+    //  Find what is causing it and fix it so this exception screen is not
+    //  displayed anymore and uncomment the test below.
+    @Test
+    public void redirectOnError() throws Exception {
+    // @checkstyle MethodBodyCommentsCheck (30 lines)
```
Comment (krzyk) [inline src/test/java/com/zerocracy/tk/TkAppTest.java]: @paulodamaso test should be ignored, not commented out

**Comment by paulodamaso** [comment]
@krzyk Done, please take a look.

**Comment by krzyk** [comment]
@paulodamaso thanks

**Comment by krzyk** [comment]
@rultor merge

**Comment by g4s8** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job was finished in 19 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @krzyk[/z](https://www.0crat.com/u/krzyk), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#1602](https://github.com/zerocracy/farm/issues/1602) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by ypshenychka** [comment]
@krzyk According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
>The code reviewer found at least three problems in the code.
Comments were mostly about design problems, not cosmetic issues.

Only one issue was found during code review. 
Please confirm that you'll try to find at least three major problems while future reviews.

**Comment by krzyk** [comment]
@ypshenychka I confirm

**Comment by 0crat** [comment]
Order was finished, quality is "acceptable": +20 point(s) just awarded to @krzyk[/z](https://www.0crat.com/u/krzyk)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Pull request merged**